db/mysql/schema.sql

```diff
@@ -438,6 +438,8 @@ ADD FOREIGN KEY (`user_id`)
     REFERENCES `usr_user` (`id`)
 	ON UPDATE CASCADE ON DELETE CASCADE;
 
+ALTER TABLE ppl_model ADD UNIQUE (name(255));
```

Comment:
Ok, so we agreed, we will leave 255 there, but with a comment why 255 should be there and how this can be improved.